<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCoolDownlaodClass</class>
 <widget class="QMainWindow" name="QCoolDownlaodClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCoolDownlaod</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sel_bt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>select file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_pt">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>download</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QCoolDownlaod.qrc"/>
 </resources>
 <connections/>
</ui>
